sale_packaging_default/views/product_template_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L. <info@moduon.team>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<data>
+
+    <record id="product_template_form_view_inherit" model="ir.ui.view">
+        <field name="name">Simplify packaging fields</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='inventory']" position="inside">
+                <group string="Packaging Division">
```

Comment:
issue: since you're creating a new group, it can land far away from the packaging configuration. Better, put the new checkbox within the same visual group as the rest of packaging configuration:

![image](https://github.com/user-attachments/assets/77b25239-fa1d-4281-82fc-570d6c1e1c99)